manual/src/main/kotlin/com.r3.corda.finance.manual/contract/DummyContract.kt

```diff
@@ -0,0 +1,8 @@
+package com.r3.corda.finance.manual.contract
+
+import net.corda.core.contracts.Contract
+import net.corda.core.transactions.LedgerTransaction
+
```

Comment:
Ah.. we need this so that corda adds the manual jar as a dependency to a transaction. Hmm.. I should really split up the ripple and swift modules into separate modules, one for types and another for flows - the flows should never be loaded by the transaction class loader.

Reply:
Yes, I got this idea for a DummyContract from the token-sdk code :)

Reply:
Ah yes :)